apps/prairielearn/src/pages/instructorStudents/components/DownloadButton.tsx

```diff
@@ -0,0 +1,85 @@
+import type { Table } from '@tanstack/react-table';
+
+import { downloadAsCSV, downloadAsJSON } from '../../../lib/client/downloads.js';
+import type { StudentRow } from '../instructorStudents.shared.js';
+
+export function DownloadButton({
+  students,
+  table,
+}: {
+  students: StudentRow[];
+  table: Table<StudentRow>;
+}) {
+  function downloadStudentsCSV(students: StudentRow[], filename: string): void {
+    const rows = students.map((student) => [
+      student.uid,
+      student.name,
+      student.email,
+      student.created_at
+        ? new Date(student.created_at).toISOString().replace('T', ' ').split('.')[0]
+        : '',
+    ]);
+    downloadAsCSV(['UID', 'Name', 'Email', 'Enrolled At'], rows, filename);
+  }
+  return (
+    <div class="btn-group">
+      <button
+        type="button"
+        data-bs-toggle="dropdown"
+        aria-expanded="false"
+        class="btn btn-outline-light btn-sm dropdown-toggle"
+      >
+        <i aria-hidden="true" class="px-2 fa fa-download"></i>
```

Comment:
Padding at the start of the icon makes the spacing unbalanced.

```suggestion
        <i aria-hidden="true" class="pe-2 fa fa-download"></i>
```

Reply:
Changed in 6bfdc21e358d030e70cbbb0b61c07a8022d3ed17.